FAQ.md

```diff
@@ -0,0 +1,105 @@
+* And a live streaming service with multibitrate (if possible).
+
+After you have your m3u8 link, add your channel information onto the [Database](https://github.com/iptv-org/database) repository, and submit an [issue](https://github.com/iptv-org/iptv/issues/new) to propose it to us.
+
+### Is there any way to add my channel other than via git repository?
+No, there is no such option.
+
+## Technical
+### Why am I asked to provide an adaptive playlist like "master.m3u8", "playlist.m3u8" or "index.m3u8"?
+These files are useful to see perfectly the channel based in your network or screen (adaptive resolution). Also the streams are permanent, without token, that avoids edit temporally their urls.
+
+### Why don't we accept XStream-Codes in streams?
+Xtream-Codes streams are, most of the times, pirated streams created by some people that can get satellite and/or PPV streams and that propose it to everyone for a fairer price. These types of servers are illegal and also highly unstable, since it depends either from the user who bought it and the server that hosts it.
+
+In short, they are poor quality streams created by unauthorized people.
+
+Based on [#5401](https://github.com/iptv-org/iptv/pull/5401).
+
+### Is possible to add streams from Youtube, Dailymotion or Twitch?
+Yes. Due to technical limitations is necessary add a [Streamlink](https://streamlink.github.io/) server link to view the content.
+
+### Is it possible to add audio-only streams, like FM radio?
+No. Exceptions are the visual radios, in which a video and audio are shown at the same time.
+
+### Why are there some call sign (KJLA-DT1.us) as ids and alphanumeric names as ids (Mychannel.us)?
+It's to differentiate the origin of the broadcast from the content. Call signs usually come from physical stations in the country of origin and their programming is subject to change. If the channel is not from a television station, the brand name will be used.
+
+An example of a call sign used as id is KJLA-DT1.us: [KJLA](https://en.wikipedia.org/wiki/KJLA) is a digital television station, DT is a suffix, 1 is a subchannel number and ".us" is the country code (United States). This station broadcasts the Visión Latina channel, whose id is VisionLatina.us.
+
+### There are source for verify call sign and coverage area for some TV stations?
+There are websites to consult the location and characteristics of the station. For example, [FCCData](https://fccdata.org/) for US, Canada, Mexico UK, Austrailia and Japan.
+
+### Why some streams have empty id?
+Because the stream does not have enough information available. If you know, let us know its official name, the language it broadcasts and the country where it originates.
+
+### What is a "daily update" and why is benefit for this playlist?
+A "daily update" is maintenance to check the status of the playlist operation, as well as to hide streams from the playlist if they have access problems.
```

Comment:
```suggestion
A "daily update" is a pull request made for maintenance purposes. It checks the status of the playlist operation, and reclassifies the streams internally. This will be useful on playlist recreation, as it's gonna hide streams from the playlist if they have access problems.
```